types/office-js-preview/index.d.ts

```diff
@@ -117993,254 +117993,248 @@ declare namespace OneNote {
          *
+         * @remarks
          * [Api set:  1.1]
          */
         document: Visio.Document;
         /**
-         *
          * Gets the success or failure of the DataRefreshComplete event.
          *
+         * @remarks
          * [Api set:  1.1]
          */
         success: boolean;
     }
     /**
-     *
      * Provides information about the shape collection that raised the SelectionChanged event.
      *
+     * @remarks
      * [Api set:  1.1]
      */
     interface SelectionChangedEventArgs {
         /**
-         *
          * Gets the name of the page which has the ShapeCollection object that raised the SelectionChanged event.
          *
+         * @remarks
          * [Api set:  1.1]
          */
         pageName: string;
         /**
-         *
          * Gets the array of shape names that raised the SelectionChanged event.
          *
+         * @remarks
          * [Api set:  1.1]
          */
         shapeNames: string[];
     }
     /**
-     *
      * Provides information about the success or failure of the DocumentLoadComplete event.
      *
+     * @remarks
      * [Api set:  1.1]
      */
     interface DocumentLoadCompleteEventArgs {
         /**
-         *
          * Gets the success or failure of the DocumentLoadComplete event.
          *
+         * @remarks
          * [Api set:  1.1]
          */
         success: boolean;
     }
     /**
-     *
      * Provides information about the page that raised the PageRenderComplete event.
      *
+     * @remarks
      * [Api set:  1.1]
      */
     interface PageRenderCompleteEventArgs {
         /**
-         *
          * Gets the name of the page that raised the PageLoad event.
          *
+         * @remarks
          * [Api set:  1.1]
          */
         pageName: string;
         /**
-         *
          * Gets the success/failure of the PageRender event.
          *
+         * @remarks
          * [Api set:  1.1]
          */
         success: boolean;
     }
     /**
-     *
      * Provides information about DocumentError event
      *
+     * @remarks
      * [Api set:  1.1]
      */
     interface DocumentErrorEventArgs {
         /**
-         *
          * Visio Error code
          *
+         * @remarks
          * [Api set:  1.1]
          */
         errorCode: number;
         /**
-         *
          * Message about error that occured
          *
+         * @remarks
          * [Api set:  1.1]
          */
         errorMessage: string;
         /**
-         *
          * Tells if the error is critical or not. If critical the session cannot continue.
          *
+         * @remarks
          * [Api set:  1.1]
          */
         isCritical: boolean;
     }
     /**
-     *
      * Provides information about the TaskPaneStateChanged event.
      *
+     * @remarks
      * [Api set:  1.1]
      */
     interface TaskPaneStateChangedEventArgs {
         /**
-         *
          * Current state of the taskpane
```

Comment:
Should be "task pane" - here and other instances

```suggestion
         * Current state of the task pane.
```